packages/flutter/lib/src/widgets/animated_child_switcher.dart

```diff
@@ -39,10 +82,14 @@ class AnimatedChildSwitcher extends StatefulWidget {
     this.child,
```

Comment:
constructor docs are out of date

also constructor docs should say what it does

Reply:
Done